Common/Sources/Insights/HyperwalletInsights.swift

```diff
@@ -84,20 +78,13 @@ public class HyperwalletInsights: HyperwalletInsightsProtocol {
         }
     }
 
-    /// Track Error
-    ///
-    /// - Parameters:
-    ///   - pageName: Name of the page - example : transfer-method:add:select-transfer-method
-    ///   - pageGroup: Page group name - example : transfer-method
-    public func trackError(pageName: String, pageGroup: String) {
+    public func trackError(pageName: String, pageGroup: String, errorInfo: ErrorInfo) {
```

Comment:
Remember to add tests for this method in `HyperwalletInsightsTests`

Reply:
done